src/reducers/rootReducer.js

```diff
@@ -19,6 +19,19 @@ const initState = {
 };
 
 const rootReducer = (state = initState, action) => {
+  if (action.type === 'ADD_EVENT') {
```

Comment:
please create file with types(const), and in this file add
`export const ADD_EVENT = 'ADD_EVENT'` and use this const here
and consider using switch in reducers